src/linode_object_storage_module.js

```diff
@@ -0,0 +1,57 @@
+import uploadFileToBucket from './aws_s3_client_module.js';
+import makeRequest from './requests.js';
+
+const BASE_LINODE_URL_SUFFIX = '.linodeobjects.com';
+const BASE_LINODE_OBJECT_URL = 'https://api.linode.com/v4/object-storage/buckets/';
+
+/**
+ * Linode Helper Module to upload a file to object storage. The file is uploaded using
+ * AWS S3 Client. Please refer to https://docs.aws.amazon.com/sdk-for-javascript/index.html for more details.
+ *
+ * @param accessKeyId bucket specific unique identifier required for authentication
+ * @param secretAccessKey user specific unique identifier required for authentication
+ * @param region indicates the geographical server location (e.g us-east-1, eu-west-1a)
+ * @param file complete path of the file to be uploaded is passed on as a parameter
+ * @param bucket uniquely identifies the bucket where the file should be uploaded
+ * @returns {Promise<void>}
+ */
+async function uploadFileToLinodeBucket (accessKeyId, secretAccessKey, region, file, bucket) {
```

Comment:
Would be nice to have some predefined constants for regions

Reply:
Sure, makes sense, created an issue to track this https://github.com/aicore/linode-object-storage-lib/issues/3